netbox/templates/_base.html

```diff
@@ -50,7 +50,16 @@ <h4><i class="fa fa-exclamation-triangle"></i> Maintenance Mode</h4>
         <div class="container">
             <div class="row">
                 <div class="col-xs-4">
-                    <p class="text-muted">{{ settings.HOSTNAME }} (v{{ settings.VERSION }})</p>
+                    <p class="text-muted">
+                        {{ settings.HOSTNAME }} (v{{ settings.VERSION }})
+                        {% if latest_version %}
+                            {% if latest_version_url %}<a target="_blank" href="{{ latest_version_url }}">{% endif %}
+                            <span class="label label-default" style="color: #ffffff; background-color: #ff9800">
+                                New version: {{ latest_version }}
+                            </span>
+                            {% if latest_version_url %}</a>{% endif %}
+                        {% endif %}
+                    </p>
```

Comment:
I'm not sure it makes sense to include this on _every_ page. I would think a notification on the main page would be sufficient. It would also be helpful to include the date and time at which the latest version was last fetched.

It might also be appropriate to display the message only to administrative (`is_staff` or `is_superuser`) users.